sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventData.java

```diff
@@ -31,26 +31,14 @@
 import static java.nio.charset.StandardCharsets.UTF_8;
 
 /**
- * The data structure encapsulating the event being sent-to and received-from Event Hubs. Each Event Hub partition can
- * be visualized as a stream of {@link EventData}. This class is not thread-safe.
- *
- * <p>
- * Here's how AMQP message sections map to {@link EventData}. For reference, the specification can be found here:
```

Comment:
is this documentation invalid / not helpful ?

Reply:
Not really because this is abstracted from them.